### PR TITLE
feat: Adds and refactors for Triton FIL examples

### PR DESCRIPTION
**Pull request by rafvasq**
Related to https://github.com/kserve/modelmesh-serving/pull/485 and https://github.com/kserve/modelmesh-serving/issues/185, this PR expands on `lightgbm` and `xgboost` examples to show that they can be deployed with Triton (in addition to MLServer).

### CODE REVIEW

File: lightgbm/lightgbm-fil/config.pbtxt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+parameters [
+  {
+    key: "model_type"
+    value: { string_value: "lightgbm" }
+  },
+  {
+    key: "output_class"
+    value: { string_value: "false" }
+  }
+]
```
Comment (ckadner) [inline lightgbm/lightgbm-fil/config.pbtxt]: "No newline at end of file" ⛔

File: xgboost/mushroom-fil/config.pbtxt [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+parameters [
+  {
+    key: "model_type"
+    value: { string_value: "xgboost_json" }
+  },
+  {
+    key: "output_class"
+    value: { string_value: "false" }
+  }
+]
```
Comment (ckadner) [inline xgboost/mushroom-fil/config.pbtxt]: "No newline at end of file" ⛔

**Review by ckadner — CHANGES_REQUESTED** [review]
Thanks @rafvasq -- just 2 little nits

File: xgboost/mushroom-fil/config.pbtxt (hunk 2 above)
Comment (rafvasq) [reply xgboost/mushroom-fil/config.pbtxt]: Thanks Christian, I took care of these.

**Review by ckadner — APPROVED** [review]
/lgtm

**Review by ckadner — APPROVED** [review]
/lgtm
/approve

**Comment by ckadner** [comment]
Thanks @rafvasq

**Pull request merged**